app/server_test.go

```diff
@@ -32,11 +32,23 @@ import (
 	"github.com/mattermost/mattermost-server/v5/utils/fileutils"
 )
 
+func newServerWithConfig(f func(cfg *model.Config)) (*Server, error) {
+	configStore, _ := config.NewMemoryStore()
+	store, _ := config.NewStoreFromBacking(configStore, nil, false)
```

Comment:
I wonder if it makes sense to ignore error here? What if something added in the future that could result an error in `NewStoreFromBacking`? I see that is commonly used like this, but like to hear your opinions.

Reply:
No harm in improving the code :+1: